src/app/select/select-container/select-container.component.unit.spec.ts

```diff
@@ -19,7 +21,39 @@ describe('MzSelectContainerComponent:unit', () => {
     fixture.detectChanges();
   });
 
-  it('should create', () => {
-    expect(component).toBeTruthy();
+  describe('ngOnDestroy', () => {
+
+    it('should remove subscriptions correctly', () => {
+      const mockSelectDirective = <MzSelectDirective>{
+        onUpdate: { unsubscribe: () => null },
+        inputElement: { off: () => null },
+      };
+
+      const mockStatusChangesSubscription = <Subscription>{
+        unsubscribe: () => null,
+      };
+
+      const mockSelectValueSubscription = <Subscription>{
+        unsubscribe: () => null,
+      };
+
+      component.mzSelectDirective = mockSelectDirective;
+      component.statusChangesSubscription = mockStatusChangesSubscription;
+      component.selectValueSubscription = mockSelectValueSubscription;
+
+      spyOn(mockSelectDirective.onUpdate, 'unsubscribe');
+      spyOn(mockSelectDirective.inputElement, 'off');
+
+      spyOn(component.statusChangesSubscription, 'unsubscribe');
+      spyOn(component.selectValueSubscription, 'unsubscribe');
+
+      component.ngOnDestroy();
```

Comment:
I think it would be better if we test that the subscriptions are closed instead of expecting the method have been called.

Reply:
Done